docs/index.txt

```diff
@@ -72,8 +72,41 @@ The ``jdbc:`` prefix is optional. For example::
     crate://localhost:4300
 
 
+Compatibility
+=============
+
+JDBC
+----
+
+This JDBC driver complies to the `JDBC 4.1`_ standard as good as possible and reasonable for
```

Comment:
I'd rephrase that to:
`This JDBC driver complies to the `JDBC 4.1`_ standard to a very good level but not fully. The following mandatory features are not supported:`